CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## 2.8.5 - 
+
+feat(Typescript): Migrated the SDK to TypeScript for better type safety and improved development experience.
+- Added TypeScript definitions for all modules and functions in the SDK.
+- Added comments throughout the codebase to improve readability and maintainability.
+- Added a type declarations file (*.d.ts) to provide better type checking and editor support for consumers of the SDK.
+- Improved the build process to incorporate the TypeScript compiler and generate both CommonJS and ES module versions of the SDK.
```

Comment:
@sidag95 Are we also releasing ES modules? This is cool.

Reply:
No this is not supported. @ankitdas13 Lets remove ES module version. We are not realsing it in ES modules. Let's remove this line.